### ISSUE TITLE
Upgrade slash-commands support

### ISSUE THREAD

**Issue by ghost** [v8-riscv/v8-riscv-ci#6]
Currently, the tool checks for the whole comment body to match the command. It may be more convenient to check for the command to show up anywhere in the body of the comment.

**Comment by lazyparser** [comment]
It may be a good idea to detect the first char of every line, leaving other slash strings untouched.

e.g.

```
/slash parse it
comment /do-not-parse
```

**Comment by ghost** [comment]
That's a good idea. Then we can just use the rest of that line for arguments to the command. So it could be:
```
/retest precheck
/retest all
/retest build run
```

**Comment by lazyparser** [comment]
Yes. That's exactly the way other slash commands  utility/tools does.

After you pushed the initial codes, we can add more sub commands :)

**Comment by lazyparser** [comment]
and the `/retest` should be the alias to `/retest all` :)

**Comment by lazyparser** [comment]
I've read the code. It's clean and clear :)
I'll continue to add new features if needed.
